web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1582,6 +1584,10 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
 
   private def logSnippetFailure(sf: SnippetFailure) = logger.info("Snippet Failure: " + sf)
 
+  val throwSettingsException: LiftRulesSettingException => Unit = e => throw e
+  val warnOnSettingsException: LiftRulesSettingException => Unit = e => logger.warn("LiftRules setting safety violation!!!", e)
```

Comment:
Ditto to the above comment. This is consistent with what we do with other error handling behavior.